FastSurferCNN/data_loader/conform.py

```diff
@@ -107,16 +129,42 @@ def options_parse():
     advanced.add_argument(
         "--conform_to_1mm_threshold",
         type=__conform_to_one_mm,
-        help="Advanced option to change the threshold beyond which images are conformed to 1"
-        "(default: infinity, all images are conformed to their minimum voxel size).",
+        help="Advanced option to change the threshold beyond which images are "
+             "conformed to 1 (default: infinity, all images are conformed to their "
+             "minimum voxel size).",
     )
-    parser.add_argument(
+    advanced.add_argument(
         "--dtype",
         dest="dtype",
         default="uint8",
         type=__target_dtype,
-        help="Specifies the target data type of the target image or 'any' (default: 'uint8', "
-        "as in FreeSurfer)",
+        help="Specifies the target data type of the target image or 'any' (default: "
+             "'uint8', as in FreeSurfer)",
+    )
+    advanced.add_argument(
+        "--no_strict_lia",
+        dest="force_strict_lia",
+        action="store_false",
+        help="Ignore the forced lia reorientation.",
+    )
+    advanced.add_argument(
+        "--no_lia",
+        dest="force_lia",
+        action="store_false",
+        help="Ignore the reordering of data into lia (without interpolation). "
+             "Superceeds --no_strict_lia",
+    )
+    advanced.add_argument(
+        "--no_vox_size",
+        dest="force_vox_size",
+        action="store_false",
+        help="Ignore the forced isometric voxel size (depends on --conform_min).",
+    )
+    advanced.add_argument(
+        "--no_img_size",
```

Comment:
why not no_img_dim ?

Reply:
dim is usually the number of dimensions, so usually 3 or 4 in python (numpy/torch)
shape would be what we mean, but I am not sure it that is more "helpful"

Reply:
yes, I think size is fine